OJPP.js

```diff
@@ -178,14 +182,29 @@ async function izpisi_zamudo(gumb, busId) {
         opacity: 0.7">${z.zamuda} min</span>&nbsp${z.postaja}</li>`;
     }
     zamudeHTML += "</ul>";
+
+    //Update the button text
+    gumb.innerText = "Pokaži vse zamude";
+    gumb.onclick = () => {
+        izpisi_zamudo(gumb, busId, 0);
+    }
     zamudiceContainer.innerHTML = zamudeHTML;
+
 }
 
 TIMETABLE = document.getElementById("timetable");
-function izpisi_urnik(trips) {
+async function izpisi_urnik(trips) {
     TIMETABLE.innerHTML = "<thead><tr><td>Ura</td><td>Linija</td><td>Trajanje</td><td>Prevoznik</td></tr></thead>";
     for(let t of trips) {
 
+        console.log(t);
+        
+        var res;
+        result = await checkForDeletedElement(`https://ojpp.si/trips/${t?.["trip_id"]}`);
```

Comment:
This looks like it would take a really long time to get through (for example, Ljubljana AP has probably hundreds of trips to certain destinations, especially given it includes deleted ones). 
If each request takes 1 second to finish, this makes the user wait for more than a minute to show the timetables.

I would suggest that all the trips be initially shown, then start hiding them according to their `deleted` status.